docs/source/developer/resources.rst

```diff
@@ -146,3 +146,144 @@ Thumbnail image
 ------------------------------------------------------------------------------
 
 This image should represent the resource, and be at least 350px high.
+
+Dynamic Text Overlay
+==============================================================================
+In many cases, resources comprise of a base PNG image with text dynamically overlayed from within the python view, based on a users request.
+Cases where this is necessary include:
+
+- Randomly generated numbers or data
+- Text, which must be translated into the user's language
+
+While the actual text is added dynamically, the layout/colour/font/size of that text on the resource should be determined as part of the design process.
+To achieve this, we have developed a pipeline to allow designers to define these text fields in Adobe Illustrator, and export them in an SVG format.
+This information can then be used to dynamically render the required text as closely as possible to the intended design. This process is outlined in more detail below, for both developers and designers.
+
+For Designers
+------------------------------------------------------------------------------
+The following workflow has been designed for Adobe Illustrator. Currently, other graphics software is not supported.
```

Comment:
What other software has been tested? The only other main one I can think of would be Inkscape.

Reply:
I haven't tried it on any other software - it's compatibility would depend entirely on the way an SVG is exported (if even possible), and how similar its structure is to the one exported from AI. This is something that could be investigated later, if the need arose.